projects/js-packages/charts/src/components/leaderboard-chart/leaderboard-chart.tsx

```diff
@@ -0,0 +1,237 @@
+import clsx from 'clsx';
+import { type FC } from 'react';
+import { formatMetricValue } from '../shared/format-metric-value';
+import styles from './leaderboard-chart.module.scss';
+
+export interface LeaderboardEntry {
+	/**
+	 * Unique internal key (e.g., 'key-direct')
+	 */
+	id: string;
+
+	/**
+	 * Human-readable name (e.g., 'Direct')
+	 */
+	label: string;
+
+	/**
+	 * Value of the entry
+	 */
+	currentValue: number;
+
+	/**
+	 * Value of the entry in the previous period
+	 */
+	previousValue: number;
```

Comment:
The implementation is limited to handling only two entries per item. Is that correct
Do you think we should consider handling more than two entries?

This is an example of a BarChart rendering three entries per item, for instance.

<img width="293" height="320" alt="Image" src="https://github.com/user-attachments/assets/78265684-ab75-4565-af07-073a99b72494" />

Reply:
Yeah, it's the current implementation. I was aiming first to migrate the component to allow for more variation in the future.

Do we already have use cases or a design for the leaderboard chart with more than two series?